infra/Pos/Communication/Relay/Types.hs

```diff
@@ -51,3 +56,17 @@ data RelayContext = RelayContext
     { _rlyIsPropagation    :: !Bool
     , _rlyPropagationQueue :: !RelayPropagationQueue
     }
+
+makeLenses ''RelayContext
```

Comment:
`makeClassy` creates classes of this shape:

```
class HasPropagationQueue s a | s -> a
```

and I want them to be of this shape:

```
class HasPropagationQueue ctx where
```

One parameter less.